src/main/java/ga/rugal/food/core/service/impl/RestaurantServiceImpl.java

```diff
@@ -55,7 +58,8 @@ public Menu getMenuByLocation()
         Menu menu = getRandomMenu(mList);
         return menu;
     }
-
+    
+    @Override
     public List<Menu> getMenuList()
     {
         List<Restaurant> rList = getWholeList();
```

Comment:
In this phase, we should remove those kind of methods and code for the sanitation of code style.